src/ur/workday.urs

```diff
@@ -0,0 +1,145 @@
+signature AUTH = sig
+    val token : transaction (option string)
+end
+
+type instance
+val read_instance : read instance
+val show_instance : show instance
+
+(* Opaque type for all Workday IDs. *)
+type wid = string
```

Comment:
The comment and type do not agree.  Considering that I think the sensible thing is for this type to _be_ opaque, either make the type opaque or change the comment to describe why this is not possible.